requirements/extras.txt

```diff
@@ -0,0 +1,5 @@
+ovos-backend-client>=0.1.0a7
+ovos-microphone-plugin-alsa~=0.0.0
+ovos-stt-plugin-server~=0.0.3
+ovos-ww-plugin-precise-lite[full]~=0.1
+ovos-vad-plugin-webrtcvad~=0.0.1
```

Comment:
Want to make this one optional too, pending a new plugin based on noise threshold only like original listener

Reply:
I pulled the default from `ovos-config` default configuration; I don't have a strong preference between the 2 plugins but I think the extras should make things work without extra config

Reply:
`silero` requires `onnxruntime` or `tfruntime` *(don't remember)*, `webrtcvad` would be more simple to install.